.circleci/config.yml

```diff
@@ -0,0 +1,52 @@
+version: 2.1
+
+orbs:
+  ghpr: narrativescience/ghpr@0.0.5
+
+commands:
+  install-deps:
+    description: Install dependencies and initialize pre-commit hooks
+    steps:
+      - run:
+          command: |
+            pip install pre-commit tox flit
+            pre-commit install
+          name: Install dependencies
+
+jobs:
+  test:
+    docker:
+      - image: circleci/python:3.7
+    steps:
+      - ghpr/build-prospective-branch
+      - install-deps
+      - run: pre-commit run --show-diff-on-failure
```

Comment:
I think by default this will only run tests against the previous commit (not the base branch) which is why we were providing the branch options. @ns-ckao correct me if im wrong though

Reply:
Ah got it, I assumed it would use the prospective branch I build in the first step. Will update